### PR TITLE
Fix master

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by solivier** [comment]
GTM

**Comment by jmleroux** [comment]
GTM

File: src/Pim/Bundle/CatalogBundle/Resources/config/storage_driver/doctrine/mongodb-odm.yml [hunk 1]
```diff
@@ -90,7 +90,6 @@ services:
         class: %pim_catalog.doctrine.naming_utility.class%
         arguments:
             - '@pim_catalog.doctrine.smart_manager_registry'
-            - '@pim_catalog.repository.locale'
```
Comment (grena) [inline src/Pim/Bundle/CatalogBundle/Resources/config/storage_driver/doctrine/mongodb-odm.yml]: BC?

Comment (solivier) [inline src/Pim/Bundle/CatalogBundle/Resources/config/storage_driver/doctrine/mongodb-odm.yml]: no it's not, I just removed that line I changed on friday

Comment (grena) [inline src/Pim/Bundle/CatalogBundle/Resources/config/storage_driver/doctrine/mongodb-odm.yml]: Ok :+1:

**Pull request merged**